app/services/food_data_service.rb

```diff
@@ -0,0 +1,7 @@
+class FoodDataService
+  def self.search(query)
+    url = 'https://api.nal.usda.gov/fdc/v1/foods/search'
+    res = Faraday.get(url, { api_key: ENV['FOOD_API_KEY'], query: query })
```

Comment:
I would prefer to see the query params set in a block. This makes the request more readable/maintainable.